### PR TITLE
feat(/src/models): new Polygon model added

### PR DESCRIPTION
**Pull request by FacundoPucheta**
New Polygon model added.

### CODE REVIEW

File: src/models/Polygon.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const { DataTypes } = require('sequelize');
+
+
+module.exports = (sequelize) => {
+    sequelize.define('Polygon', {
+        id: {
+            type: DataTypes.BIGINT(11),
```
Comment (manduinca) [inline src/models/Polygon.js]: Why not integer?

File: src/models/Polygon.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        points: {
+            type: DataTypes.STRING(2860),
+            defaultValue: null
+        },
+        type: {
+            type: DataTypes.BIGINT(11),
+            allowNull: false
+        }
+    },
+        {
```
Comment (manduinca) [inline src/models/Polygon.js]: For a correct indent it should be four back space. Right?

**Review by manduinca — DISMISSED** [review]
Maybe it can be improved. But I approved.

File: src/models/Polygon.js (hunk 2 above)
Comment (FacundoPucheta) [reply src/models/Polygon.js:21]: Can i improve the code and rebase it?

**Review by SebaSatelier — APPROVED** [review]
looks fine

**Review by manduinca — APPROVED** [review]
LGTM!

**Pull request merged**